parsers/ONS.py

```diff
@@ -192,13 +156,13 @@ def fetch_region_exchange(
     print(fetch_exchange("BR-S", "AR"))
 
     print("fetch_region_exchange(BR-CS->BR-S)")
-    print(fetch_region_exchange("BR-CS", "BR-S"))
+    print(fetch_exchange("BR-CS", "BR-S"))
 
     print("fetch_region_exchange(BR-CS->BR-NE)")
-    print(fetch_region_exchange("BR-CS", "BR-NE"))
+    print(fetch_exchange("BR-CS", "BR-NE"))
 
     print("fetch_region_exchange(BR-CS->BR-N)")
-    print(fetch_region_exchange("BR-CS", "BR-N"))
+    print(fetch_exchange("BR-CS", "BR-N"))
 
     print("fetch_region_exchange(BR-N->BR-NE)")
-    print(fetch_region_exchange("BR-N", "BR-NE"))
+    print(fetch_exchange("BR-N", "BR-NE"))
```

Comment:
It also means these need to be updated though to not introduce any type errors in the file. (the print statements containing the title should also be updated).